<launch>
	<node name="sampleimugpspublisher2SVY21" pkg="epochmini" type="sampleimugpspublisher2SVY21.py" />

	<node name="nofilter_pose_publisher" pkg="epochmini" type="nofilter_pose_publisher" />

	<node name="goalpathpublisher" pkg="epochmini" type="goalpathpublisher" />

	<node name="tf_worldtobase_goal" pkg="epochmini" type="tf_worldtobase_goal" />

	<node name="purepursuit1" pkg="epochmini" type="purepursuit1" />

	<node name="simplepid2" pkg="epochmini" type="simplepid2" />

	<node name="rosserial_arduino" pkg="rosserial_python" type="serial_node.py" args="/dev/ttyACM1" />

</launch>